warehouse/postgres/postgres_test.go

```diff
@@ -51,6 +53,13 @@ func (*TestHandle) VerifyConnection() error {
 }
 
 func TestPostgresIntegration(t *testing.T) {
+	// Setting up the test configuration
```

Comment:
Let's move this to a separate file such as warehouse_integration.test because Warehouse.enableExcludedSchema will not work for other destinations since it is a global config and not related to destination specific.

Reply:
Let's keep this file as it is and move this to a separate file called warehouse_integration.test